tsconfig.json

```diff
@@ -14,8 +14,7 @@
 		// `alwaysStrict: false` is needed because otherwise esbuild will add `"use strict";`
 		// statements throughout the imported libraries which can break them.
 		"alwaysStrict": false,
-		"skipLibCheck": true,
-		"types": []
```

Comment:
Why this change?

Reply:
empty `types` doesnt mean anything

as for `skipLibCheck` I rather not skip type checking like this if we don't have to.

Reply:
An empty `types` means that no `@types` packages will be automatically included, I think? Whereas _no_ `types` will include them automatically, which we probably don't want to do (I believe this was causing issues in the past with `@types/node` being used in the wrong places cc @mrbbot )